draft-wood-tls-ticketrequest.md

```diff
@@ -106,83 +105,56 @@ if early data is not needed by these connections, this method SHOULD NOT be used
-to clients. Its value MUST be empty if the corresponding TicketRequest context field is empty.
-
-Servers SHOULD place a limit on the number of tickets they are willing to vend to clients. Servers
-MUST NOT send more than 255 tickets to clients, as this is the limit imposed by the request and 
-response identifier size. Servers SHOULD NOT send unsolicited NewSessionTickets to clients 
-that express support for TicketRequests. 
+Clients may send this extension in ClientHello. It contains the following structure:
 
-When operated over an unreliable transport, e.g., DTLS, servers may not always be able to 
-identify retransmissions of ticket requests. In this case, when a server S receives a TicketRequest 
-with new identifier N it MUST generate a new ticket and SHOULD cache it locally for some period of 
-time T. If S receives a TicketRequest with identifier N within time period T, S SHOULD reply with 
-the same ticket previously generated (and cached). If S receives a TicketRequest with identifier 
-N outside time period T, S SHOULD reply with an empty NewSessionTicket, i.e., a NewSessionTicket 
-with extension ticket_identifier carrying N, appropriate ticket_context extension, and empty ticket 
-field.
+~~~
+struct {
+    uint8 count;
+} TicketRequestContents;
+~~~
 
-<!-- TicketRequest messages MUST NOT be sent until after the TLS handshake is complete.  -->
-<!-- As handshake messages, these MUST be added to the handshake transcript. -->
+count
+: The number of tickets desired by the client.
 
-# Negotiation 
+A supporting server MAY vend TicketRequestContents.count NewSessionTicket messages to a
+requesting client, and SHOULD NOT send more than TicketRequestContents.count NewSessionTicket
+messages to a requesting client. Servers SHOULD place a limit on the number of tickets they are willing to
+vend to clients. Thus, the number of NewSessionTicket messages sent should be the minimum of
+the server's self-imposed limit and TicketRequestContents.count. Servers MUST NOT send more
+than 255 tickets to clients, as this is the limit imposed by TicketRequestContents.count.
 
-Clients negotiate use of ticket requests via a new ExtensionType, ticket_request(TBD). 
-The extension_data for this extension MUST be empty, i.e., have length of 0. Servers that support ticket 
-requests MAY echo this extension in the EncryptedExtensions. Clients MUST NOT send ticket requests to servers
-that do not signal support for this message. If absent from a ClientHello, servers MUST NOT generate 
-responses to TicketRequests issued by the client.
+Servers that support ticket requests MAY echo "ticket_request" in the EncryptedExtensions.
 
 # IANA Considerations
 
-((TODO: codepoint for post-handshake message type and extensions))
+IANA is requested to Create an entry, ticket_requests(0xffcd), in the existing registry
+for ExtensionType (defined in {{RFC8446}}), with "TLS 1.3" column values being set to
+"CH, EE", and "Recommended" column being set to "Yes".
```

Comment:
Is this the standard practice for adding new items, or does the table get drawn out ever?

Reply:
It's what we're using in the ESNI draft, so, I guess so?

Reply:
Okay, sounds good then!